gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagActionStore.java

```diff
@@ -45,6 +47,7 @@ class DagAction {
     final String flowExecutionId;
     final String jobName;
     final DagActionType dagActionType;
+    boolean isReminder = false;
```

Comment:
to differentiate a triggered dag action from a natural dag action in dag task stream impl.